src/data/lancer/lcp/homebrew/ironleaf-foundry/frames.json

```diff
@@ -580,8 +580,8 @@
   {
     "id": "mf_scilla_alt_laurel",
     "source": "IF",
-    "name": "LAUREL",
-    "variant": "SCILLA",
+    "name": "LaurelL",
```

Comment:
is this how it is in the LCP? `LaurelL`? I can't imagine it is. Even if it is, let's fix it to just `Laurel`, matching the other caps lock to normal cased changes